druid/examples/flex.rs

```diff
@@ -323,7 +323,8 @@ fn make_ui() -> impl Widget<AppState> {
 
 fn main() -> Result<(), PlatformError> {
     let main_window = WindowDesc::new(make_ui)
-        .window_size((600., 600.00))
+        .window_size((620., 265.00))
```

Comment:
```suggestion
        .window_size((620., 600.00))
```
I think it's clearer when these are different, and we currently can't display the  column example  at our initial size.